test/common/test_md_index.cpp

```diff
@@ -57,11 +57,11 @@ int main() {
 
     "md_index_flat"_test = [] () {
         expect(eq(
-            GridFormat::flat_index(MDIndex{{0}}, GridFormat::MDLayout{std::vector{1}}),
```

Comment:
I believe I put these braces because with the older compilers tested in the pipeline this didn't work. Let's try, though.